packages/module/src/catalog/QuickStartTile.tsx

```diff
@@ -17,13 +17,15 @@ interface QuickStartTileProps {
   status: QuickStartStatus;
   isActive: boolean;
   onClick?: () => void;
+  action?: React.ReactNode;
```

Comment:
Wondering if should just allow a callback function instead and build the bookmark button ourself inside.  I got some strange results passing in the incorrect type of button.

Reply:
If we are not going to build it internally, we need documentation for how to use these props and this component. At the very least, we should add comments to each of these `QuickStartTileProps`. Eventually, those props can be used in the quickstarts docs on PF.org

I'm noticing that there is no example in the PF docs for how to build a `CustomCatalog` as is done in the demo app. ideally, we'd have an example that would also expose the APIs for all these components consumers could be using. That would need to be done in a follow up issue.

Reply:
I like this approach! I've changed it to be a button with predefined config. For power users I've also allowed passing `buttonProps` should they need to tinker with the button (for example changing the hover state etc.).

@nicolethoen good point on adding this to a `CustomCatalog`, I've added another section with same content as is rest and added the bookmark toggle.